src/logic/operate.js

```diff
@@ -0,0 +1,29 @@
+import Big from 'big.js';
+
+const operate = ({ total = '', next = '', operation = '' }) => {
+  const x = new Big(total);
+  const y = new Big(next);
+  let result;
+  switch (operation) {
+    case '%':
```

Comment:
- [ ] As it said in the instructions, operate should accept 3 parameters: numberOne, numberTwo, and operator. The operate method is going to be called by the calculate method when the user inputs an operation to happen.
![Screenshot from 2021-01-12 14-01-21](https://user-images.githubusercontent.com/30977845/104346868-ac974600-54de-11eb-824e-d7b6d7beed0d.png)

- [ ] The modifier operator here is supposed to work differently from the `mod`'s Big.js. Mod here is supposed to turn the number into a percentage based number, e.g. `95 % == .95`
![Screenshot from 2021-01-12 14-08-26](https://user-images.githubusercontent.com/30977845/104347779-c38a6800-54df-11eb-84af-2fd5f798fe19.png)